packages/jest-cli/src/getChangedFilesPromise.js

```diff
@@ -24,6 +26,18 @@ export default (
       changedSince: globalConfig.changedSince,
       lastCommit: globalConfig.lastCommit,
       withAncestor: globalConfig.changedFilesWithAncestor,
+    }).catch(e => {
+      const message = formatExecError(e, configs[0], {noStackTrace: true})
+        .split('\n')
+        .filter(line => !line || !line.includes('Command failed:'))
```

Comment:
@SimenB Just noticed the first part actually _includes_ empty lines. Fixed in PR ##7117

Reply:
That's on purpose - we want to keep the newlines in the output. This is how it looks:
![image](https://user-images.githubusercontent.com/1404810/46588332-c42f2b00-ca9a-11e8-97df-afcf3c2a551f.png)

The condition is there in case `typeof line !== 'string'`, but thinking about it, that can't happen